### DISCUSSION TITLE
project table configuration (added colums)

### DISCUSSION THREAD

**Discussion by macciocu** [community/community#9387] [Projects and Issues]
Each time I open the project, I need to add the columns I want to see, e.g. I want to see milestones by default (and I also want everybody else to see this by default). Ideally I can save the project table configuration so that I don't have to change this each time.

**Comment by ChrisH-BH** [comment]
I missed this, but I see that if you click the drop down arrow next to tab name (default is `View 1`), you can save your layout changes.
